llvm/test/CodeGen/NVPTX/wgmma-commit-group.ll

```diff
@@ -0,0 +1,15 @@
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_90a -mattr=+ptx80 | FileCheck %s
+; RUN: %if ptxas-12.0 %{ llc < %s -march=nvptx64 -mcpu=sm_90a -mattr=+ptx80 | %ptxas-verify -arch=sm_90a %}
+
```

Comment:
Since all the tests are for wgmma (same sm/pxt-version), can we have them in a single file?
(May be, wgmma_sm90a_intrinsics.ll?)

Reply:
Combined the tests into one file, `wgmma_sm90a_intrinsics.ll` now.